libs/wingsdk/src/shared-aws/bucket.inflight.ts

```diff
@@ -27,13 +29,13 @@ export class BucketClient implements IBucketClient {
    * @param key Key of the object
    */
   public async exists(key: string): Promise<boolean> {
-    const command = new ListObjectsV2Command({
+    const command = new HeadObjectCommand({
```

Comment:
Nice! That's a great initiative 💫 
Can you please replace this in the tests as well? 
(`wing/libs/wingsdk/test/shared-aws/bucket.inflight.test.ts`)